Makefile

```
@@ -313,17 +314,17 @@ lint: check-makefiles
 		github.com/thanos-io/thanos/pkg/store/cache" \
 		./pkg/... ./cmd/... ./tools/... ./integration/...
 
-format:
+format: ## Run gofmt and goimports.
 	find . $(DONT_FIND) -name '*.pb.go' -prune -o -type f -name '*.go' -exec gofmt -w -s {} \;
 	find . $(DONT_FIND) -name '*.pb.go' -prune -o -type f -name '*.go' -exec goimports -w -local github.com/grafana/mimir {} \;
 
-test:
+test: ## Run all tests.
```

I would clarify it's just unit tests, because integration tests are skipped because of the missing build tag in the command we run below.

```suggestion
test: ## Run all unit tests.
```